routes.js

```diff
@@ -0,0 +1,67 @@
+const express = require('express')
+const router = express.Router()
+const low = require('lowdb')
+const fileAsync = require('lowdb/lib/storages/file-async')
+const db = low('db/db.json', {
+  storage: fileAsync
+})
+
+//CRUD Read
+router.get('/movies', (req, res) => {
```

Comment:
Because all of the routes in this router being with `/movies`, it is preferred to do `app.use("/movies", router)` in `app.js` and then each route in this file would be similar to `router.get("/", ...`